### ISSUE TITLE
Core module network/eos/eos_command broken by changes in module_utils/network

### ISSUE THREAD

**Issue by grybak-arista** [ansible/ansible#16509]
##### ISSUE TYPE

- Bug Report
##### ANSIBLE VERSION

```
ansible 2.2.0 (devel c7129c4efe) last updated 2016/06/29 11:04:32 (GMT -600)
  lib/ansible/modules/core: (detached HEAD 2f0f04437b) last updated 2016/06/29 11:04:40 (GMT -600)
  lib/ansible/modules/extras: (detached HEAD d25c487ac8) last updated 2016/06/29 11:04:45 (GMT -600)
  config file =
  configured module search path = Default w/o overrides

```
##### OS / ENVIRONMENT

Running from OS X 10.10.5 against Arista EOS 4.15.6M
##### SUMMARY

Using core module ansible/lib/ansible/modules/core/network/eos/eos_command.py results in an error from ansible/lib/ansible/module_utils/network.py that NetworkModule has no attribute 'execute'.
##### STEPS TO REPRODUCE

Execute an ansible ad-hoc command or ansible-playbook using the eos_command module, and the command fails. The specific commands passed to the module do not matter, the module is failing before the command is sent.

```
ansible test_hosts -i test/fixtures/hosts -c local -m eos_command -a "host=dut username=admin password=admin use_ssl=no authorize=yes transport=cli commands='configure terminal'" -vvvv
```

##### EXPECTED RESULTS

Running the command against v2.1.0.0-1 results in the following output:

```
172.16.131.40 | SUCCESS => {
    "changed": false,
    "invocation": {
        "module_args": {
            "auth_pass": null,
            "authorize": true,
            "commands": [
                "configure terminal"
            ],
            "host": "dut",
            "interval": 1,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": null,
            "provider": null,
            "retries": 10,
            "ssh_keyfile": null,
            "transport": "cli",
            "use_ssl": false,
            "username": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "waitfor": null
        },
        "module_name": "eos_command"
    },
    "stdout": [
        ""
    ],
    "stdout_lines": [
        [
            ""
        ]
    ]
}
```
##### ACTUAL RESULTS

eos_command module raises AttributeError: 'NetworkModule' object has no attribute 'execute'
I believe this is coming from ansible/lib/ansible/module_utils/network.py

```
No config file found; using defaults
Loaded callback minimal of type stdout, v2.0
Using module file /Users/grybak/working/virtualenvs/ansitest/ansible/lib/ansible/modules/core/network/eos/eos_command.py
<172.16.131.40> ESTABLISH LOCAL CONNECTION FOR USER: grybak
<172.16.131.40> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo $HOME/.ansible/tmp/ansible-tmp-1467234627.96-179960737464124 `" && echo ansible-tmp-1467234627.96-179960737464124="` echo $HOME/.ansible/tmp/ansible-tmp-1467234627.96-179960737464124 `" ) && sleep 0'
<172.16.131.40> PUT /var/folders/b7/n3q7cy3x35q5nyd13s82g4340000gp/T/tmpXE_InO TO /Users/grybak/.ansible/tmp/ansible-tmp-1467234627.96-179960737464124/eos_command.py
<172.16.131.40> EXEC /bin/sh -c 'chmod -R u+x /Users/grybak/.ansible/tmp/ansible-tmp-1467234627.96-179960737464124/ && sleep 0'
<172.16.131.40> EXEC /bin/sh -c '/usr/bin/python /Users/grybak/.ansible/tmp/ansible-tmp-1467234627.96-179960737464124/eos_command.py; rm -rf "/Users/grybak/.ansible/tmp/ansible-tmp-1467234627.96-179960737464124/" > /dev/null 2>&1 && sleep 0'
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/var/folders/b7/n3q7cy3x35q5nyd13s82g4340000gp/T/ansible_epAUm_/ansible_module_eos_command.py", line 182, in <module>
    main()
  File "/var/folders/b7/n3q7cy3x35q5nyd13s82g4340000gp/T/ansible_epAUm_/ansible_module_eos_command.py", line 152, in main
    response = module.execute(commands)
AttributeError: 'NetworkModule' object has no attribute 'execute'

172.16.131.40 | FAILED! => {
    "changed": false,
    "failed": true,
    "invocation": {
        "module_name": "eos_command"
    },
    "module_stderr": "Traceback (most recent call last):\n  File \"/var/folders/b7/n3q7cy3x35q5nyd13s82g4340000gp/T/ansible_epAUm_/ansible_module_eos_command.py\", line 182, in <module>\n    main()\n  File \"/var/folders/b7/n3q7cy3x35q5nyd13s82g4340000gp/T/ansible_epAUm_/ansible_module_eos_command.py\", line 152, in main\n    response = module.execute(commands)\nAttributeError: 'NetworkModule' object has no attribute 'execute'\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE",
    "parsed": false
}

```

**Comment by nitzmahone** [comment]
cc @privateip @Qalthos

**Comment by privateip** [comment]
I just merged change ansible/ansible-modules-core#4065 that fixes this problem when running from devel
